src/graph.cpp

```diff
@@ -203,6 +212,86 @@ void Graph::augmentPath(const std::string &source, const std::string &target, co
     }
 }
 
+//change to return void
+
+/**
+ * Takes a number and sets the bool "selected" of that amount of edges and the corresponding reverses to false
+ * Time Complexity: O(|VE|)
+ * @param numEdges - Number of edges to be deactivated
+ * @return A vector of pointers for all the edges that were deactivated
+ */
+std::vector<Edge*> Graph::deactivateEdges(int numEdges) {
+    int stationNum = 0;
+    Vertex* currentVertex;
+    int choice = 0;
+    std::vector<Edge*> deactivatedEdges = {};
+    while(deactivatedEdges.size() < numEdges){
+        stationNum = rand() % vertexSet.size();
+        currentVertex = findVertex(stationNum);
```

Comment:
I probably misread last time but here you should actually use vertexSet[stationNum] because you are generating a number randomly and want the vertex at that index. When when you are looking for a vertex with a specific id should you use findVertex()

Reply:
This alteration would change the complexity too